corgi/tasks/managed_services.py

```diff
@@ -93,7 +93,6 @@ def cpu_manifest_service(product_stream_id: str, service_components: list):
             root_node = ComponentNode.objects.create(
                 type=ComponentNode.ComponentNodeType.SOURCE,
                 parent=None,
-                purl=root_component.purl,
```

Comment:
We should probably remove the setting of purl attribute in other places too to keep thing consistent. I don't think we need it since ComponentNode.save() always set it the the component.purl anyway.